### ISSUE TITLE
[Idea] Use conventions to reduce boilerplate and number of keywords

### ISSUE THREAD

**Issue by Jure-BB** [LanguageDev/.NET-language-ideas#43]
Conventions build into the language itself could reduce boilerplate, number of keywords and make language more readable. 

Supporting many conventions may sound nice, but in reality this doesn't make language more capable and different conventions can make code harder to read. While no specific style will please everyone, unified style will make different codebases more familiar to everyone and make it possible to replace some of the keywords with conventions — consequently reducing boilerplate. 

An example from Dart: A leading underscore character ( _ ) indicates that a member is private. So, there is no need for a `private` keyword.

**Comment by LPeter1997** [comment]
I agree, that this can be useful for reducing features, complexity and readability.

As many things, this is a double-edged sword tho, as some of the semantic value and control is lost. We need to find a balance, and we'll definitely be able to incorporate some useful such conventions.

**Comment by Jure-BB** [comment]
I agree. This should apply only to cases where expressiveness of the language won't be limited by it.
